### DISCUSSION TITLE
installed another spacy module from python env

### DISCUSSION THREAD

**Discussion by sugizo** [explosion/spaCy#7199] [Help: Other Questions] [answered]
**objective**
can test spacy on jupyter notebbok without installed another spacy module from command line
**e.g. in nltk**
```
import nltk
nltk.download('popular')
```

**question**
is spacy have similar function ?
**e.g.**
```
import spacy
spacy.download('en_core_web_sm')
```

thanks

**Comment by adrianeboyd** [comment, marked as answer]
You can run:

```none
spacy.cli.download("en_core_web_sm")
```

or:

```none
!python -m spacy download en_core_web_sm
```

Because of how newly installed packages are detected, you'll probably need to start the runtime in the notebook to be able to load it with `spacy.load`.
